packages/cardano-services/src/ChainHistory/DbSyncChainHistory/types.ts

```diff
@@ -95,7 +95,7 @@ export interface WithdrawalModel {
 
 export interface RedeemerModel {
   index: number;
-  purpose: string;
+  purpose: 'cert' | 'mint' | 'spend' | 'reward' | 'voting' | 'proposing';
```

Comment:
:ok_hand: 

Probably it is possible to automatically get them from `RedeemerModel` but this is way better than `string`

Maybe `keysof RedeemerModel["purpose"]`

Reply:
Sorry @gytis-ivaskevicius but I don't understand what you're suggesting as the source, this is the `RedeemerModel` :thinking:

Reply:
@gytis-ivaskevicius next action on you here

Reply:
I see, that would be difficult to achieve, it's options from db-sync schema